ruby_on_rails/bootstrap.md

```diff
@@ -30,3 +36,52 @@ config.label_class = 'form-control-label'
 config.input_class = 'form-control'
 ```
 
+To make the error highlighting work you need to add some css to your application
+
+```scss
+.invalid-feedback {
```

Comment:
can you please explain me why this is necessary?

Reply:
In the `4.0.0.beta` version of bootstrap, the `invalid-feedback` class has per default the attribute `display: none`. When the input element has the class `is-invalid`, this attribute gets overridden to `display: block`. Because I haven't found a way to add this class to the input element with the simple form configs, I override the class in the css.

Reply:
is simpleform that adds `is-invalid`?

Reply:
No. The Problem is that simpleform can't add this class.